### DISCUSSION TITLE
How can i control shadow dynamicly ?

### DISCUSSION THREAD

**Discussion by mertthesamael** [mkosir/react-parallax-tilt#56] [Q&A] [answered]
Hey there, i have a problem about shadows. I want to make dynmaic shadow on parent div that follows tilted element. I cant find any solution for this one, when i give drop or box shadow, it appears like highlight, as expected. Thanks for help already.

**Comment by mertthesamael** [comment, marked as answer]
I find it this way,

Got the values from onMove (x and y axis degree)

Giving inline css to element and set the shadow X and X equal to onMove X and X angles (Multiplying the value of angle by 0.1 works fine.)

**Comment by mkosir** [comment]
Hey @mertthesamael, thanks for describing your use case and solution for it 🚀 
p.s. probably it's worth considering to include "shadow tilt" in library itself
